### DISCUSSION TITLE
Playlist Song Theme

### DISCUSSION THREAD

**Discussion by skeggia84** [AzuraCast/AzuraCast#3684] [Q&A]
Hello everyone, I state that I am new to Azuracast. I searched through the various pages but found no answer to my question. I would like to know how to insert a song theme for a broadcast, that is an mp3 file that always starts as an introduction to the playlist in the programming timetable. I managed to find the Liquidsoap folder in the DigitalOcean droplet but I don't know what and how to change in order to have this functionality. Thanks to all, if there should be a tutorial that is right for me give me the references.

**Comment by Vaalyn** [comment]
Currently it's not possible to prepend a playlist with a specific song that should always play first.

If you have a sequential playlist where you can manually define the order in which your media should play you can put the theme track in the beginning. For random or shuffled playlists (as far as I'm aware) there is currently no way to do this.

You can create a feature request for this on https://features.azuracast.com/
